all_test.go

```diff
@@ -325,6 +324,24 @@ func isPrefix(prefix, str string) bool {
 	return strings.Index(str, prefix) == 0
 }
 
+type TestNonLockableUser struct {
```

Comment:
It would be a bit less code if you just had one thing like this:
```go
type TestLockableUser struct {
    enableLocking bool
}

func (x TestLockableUser) IsLockable(identity string) (bool, error) {
	return x.enableLocking, nil
}

```